doc/pages/architecture/065-groups.md

```diff
@@ -0,0 +1,16 @@
+# Groups
+
+Astarte supports creating groups of devices in a realm.
+
+In this release, groups are mainly used to provide access control, combining Astarte's [path based
```

Comment:
Remove "in this release". All documentation is versioned.

Reply:
if you want to remark an evolving scenario, "Groups are currently useful mainly to provide [...]"